### PR TITLE
Update translations.md with new Korean contributor

### PR DESCRIPTION
**Pull request by henrif75**

### CODE REVIEW

File: src/running-the-course/translations.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ volunteers:
   [@noahdragon], [@superwhd], [@SketchK], and [@nodmp].
 - [Chinese (Traditional)][zh-TW] by [@hueich], [@victorhsieh], [@mingyc],
   [@kuanhungchen], and [@johnathan79717].
-- [Korean][ko] by [@keispace], [@jiyongp], and [@jooyunghan].
+- [Korean][ko] by [@keispace], [@jiyongp], [@jooyunghan], an [@namhyung]
```
Comment (namhyung) [inline src/running-the-course/translations.md:12]: an -> and

Comment (namhyung) [reply src/running-the-course/translations.md:12]: Also it needs a period at the end.

Comment (mgeisler) [reply src/running-the-course/translations.md:12]: Hi @namhyung, good catch! If you haven't seen it already, GitHub has a suggestion feature which makes it super easy to fix such things:

```suggestion
- [Korean][ko] by [@keispace], [@jiyongp], [@jooyunghan], and [@namhyung].
```

**Review by mgeisler — APPROVED** [review]
Thanks and welcome onboard, @namhyung!

**Pull request merged**